packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -145,15 +146,13 @@ function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDiv
     selectionBehavior: props.selectionStyle === 'highlight' ? 'replace' : 'toggle'
   });
   let layout = useListLayout(state, props.density || 'regular');
-  let keyboardDelegate = useMemo(() => new GridKeyboardDelegate({
+  let keyboardDelegate = useMemo(() => new ListGridKeyboardDelegate({
```

Comment:
Note that the `focusMode` of the gridState and the keyboard delegate is still `cell`, I think I could change it to `row` but it was just easier to go ahead and delete `onFocus` from `useGridRow` rather than from `useGridCell`. The `onFocus` from `useGridCell` has some additional logic merged in from `useSelectableItem `